lib/sdk/dartino_sdk.dart

```diff
@@ -13,6 +14,7 @@ import 'package:atom_dartino/sdk/sdk.dart';
 import 'package:atom_dartino/sdk/sdk_util.dart';
 
 import '../dartino.dart' show pluginId;
+import 'package:atom_dartino/proc.dart';
```

Comment:
Done as part of a later commit https://github.com/dartino/atom-dartino/commit/723e50a2a1e900e3a7e41fa5d9e29a6faa955e70